algolia/instantsearch.js
# fix(build): remove jsx pragma comments from build output

**Summary**



Removes `@jsx h` comments from the built outputs.



**Result**



fixes https://github.com/algolia/react-instantsearch/issues/3632

These comments aren't needed, as the jsx is already transpiled, and it can cause issues when someone uses the automatic runtime option, which doesn't allow jsx pragmas

fixed in two places
- umd / cjs / esm --> babel config
- .d.ts --> postprocessing script of the tsc

**How to test**

Compare the build of before and after this PR, and check whether the only thing which disappeared is the `/** @jsx h */`, and that there are no other instances of the pragma (written differently maybe)
